specs/specTemplate/README.md

```diff
@@ -102,15 +103,15 @@ export class ComponentsDemo extends React.Component<{}, {}>{
 
 ## Style API
 
-### Subcomponents (pseudo-elements)
+#### Subcomponents / Children of the Component (pseudo-elements)
```

Comment:
not yet, as I still don't understand what it does. "slot" is now the preferred term in W3C Web Components (used to be <content> but, AFAICT, it does a different thing (but I may be totally wrong, see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot). Chat tomorrow (Wed) at my weekly?